_docs/_latest/develop/dev/setup.md

```diff
@@ -153,6 +153,14 @@ try the following:
 
 ``sudo apt install default-libmysqlclient-dev``
 
+------------------
+
+If on Linux executing ``workon forseti-security``
+results in ``bash: workon: command not found`` then
+check whether workon is in the source path. Try:
```

Comment:
The `source` command actually runs the `virtualenvwrapper.sh` in the current shell.  So, it would be better to reword `check`, to something like:

```
....then ensure workon is in the source path
```